content/md/permissions-settings/manage-the-interface-and-actions-accesses.md

```diff
@@ -23,26 +23,40 @@ For almost each screen and button in the PIM, you can say if a user can see/use
 
 # Rights on products pages
 
+## Permissions on products
+
 | Permission name | Effect on the interface |
 |---------------------------|--------------|
 | Add product to groups                          | User can add products to group, in `Settings`/`Groups` he will see the `Products` tab               |
-| View the association types of a product        | User can see the `Associations` menu entry in the product form                                  |
+| View the associations of a product        | User can see the `Associations` menu entry in the product form  |
+| Add associations to a product (_since 2.1_)         | User can add associations in the `Associations` tab using the `Add associations` button  |
+| Remove associations to a product (_since 2.1_)       | User can remove an association in the `Associations` tab using the cross                               |
 | Consult the categories of a product            | User can see the `Categories` menu entry in the product form                                    |
 | Change product Family                          | User can change the product’s family in the panel on the left of the product form      |
 | Change state of product     | User can change the status of the product: `enable` or `disable` directly in the panel on the left of the product form  |
 | Comment products | User can see the `Comments` menu entry in the product form |
-| Create a product                               | User can create a new product in the product grid through the `Create product` button             |
+| Create a product                               | User can create a new product in the product grid through the `Create` button             |
 | Download the product as PDF                | User will be able to generate a PDF file containing all the product information using the `PDF` button in the `...` dropdown on the top of the product form |
 | Edit attributes of a product                   | User can edit the product’s attributes under the `Attributes` menu entry of the product form   |
 | View product history | User can see the `History` menu entry in the product form  |
 | List products                                  | User can see the list of products in several pages: on the product grid in `Products`, as well as on the `Associations` menu entry of the product form, in `Groups`... |
 | Product mass edit actions | User can make mass actions on the product grid |
-| Remove a product           | User can remove a product from the PIM using the `Delete` button in the `...` dropdown on the top of the product form      |
+| Remove a product           | User can remove a product from the PIM using the `Delete` button in the `...` dropdown on the top of the product form or the product grid (trash icon)      |
```

Comment:
`or using directly the trash icon in the product grid`

Reply:
done